packages/url_launcher/url_launcher_macos/macos/Classes/UrlLauncherPlugin.swift

```diff
@@ -5,7 +5,26 @@
 import FlutterMacOS
 import Foundation
 
+/// A handler that can launch other apps, check if any app is able to open the URL.
+public protocol SystemURLHandler {
+
+  /// Opens the location at the specified URL.
```

Comment:
This needs to document the return value. https://google.github.io/swift/#parameter-returns-and-throws-tags

Reply:
Added.